jobboard/migrations/0003_alter_recruiter_company_id.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.9 on 2021-11-12 15:23
```

Comment:
It seems that you've created the models, made the migrations, and them made changes and created another migration for those changes.

Since everything ends up going in as a single PR - its better to clean this up and have a single, clean migration file.